docs/advanced-config.md

```diff
@@ -75,3 +75,30 @@ The following config options can be set by passing them as arguments to `startTr
    - `node_version`: `process.versions.node`, e.g. `16.10.0`
 
 \*: Overwritten default value
+
+### Start all
```

Comment:
I think this should become the go-to API, perhaps move this to the top and name the section e.g. `Start instrumentation`?

Reply:
I agree, but since this feature is additive I don't think we need to do it right now.